src/app/components/outcomes-model/outcomes-model.component.responsive.css

```
@@ -0,0 +1,529 @@
+  }
+}
+
+/* =========================================================
+   TABLET
+   ========================================================= */
+
+@media (max-width: 58.75rem) {
+
+  .outcomes-model {
+    --side: 2rem;
+    font-size: 1rem;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)